articles/cloud-storage/cs-how-use-file-browsers.md

```diff
@@ -146,78 +146,64 @@ To install and configure S3 Browser:
 
-To install and configure Atmos for Chrome:
+### Atmos HTML5 Application
 
-1. Download and install the [Atmos for Chrome extension](https://chrome.google.com/webstore/detail/atmos-for-chrome/befofpjhdclgccimklkaheifjajbcmnk).
+   > [!NOTE]
+   > For installation and initial configuration of the Atmos HTML5 application, you will need the following Prerequisite information:
+   > Atmos Access Key - in the form of SubtennantID/User
+   > Atmos Secret Key - Shared Secret Key
+   > A folder within an existing bucket or new bucket to host the GUI files e.g. /<bucket_name>/HTML5UI or /HTML5UI
 
-2. After you complete the installation, you'll see a globe icon in your browser toolbar. Click this icon to launch the extension.
+To install and configure the Atmos HTML5 application on windows:
 
-    ![Atmos for Chrome toolbar icon](images/cs-file-browsers-ext-1.png)
+1. Download and extract the [Atmos HTML5 application](INSERT URL HERE).
 
-3. To connect the extension to Cloud Storage, click the **Manage Accounts** button.
+2. Open a command prompt and navigate to the extracted files and run the following command: upload.bat WebContent\main.html /<bucket_name>/<optional_path>/
 
-4. Add an account for Cloud Storage using your credentials, which you can find in the *Storage* section of the UKCloud Portal.
+3. The installer script will run and prompt for the following information.
 
-    > [!NOTE]
-    > The UID value is a combination of the subtenant ID (bucket name) and your user ID, joined by a forward slash (`/`): `<subtenant_id>/<user_id>`. Ensure you enter the correct endpoint in the **Atmos URL** field, as shown in [API endpoints](#api-endpoints).
-
-    ![Atmos for Chrome Manage Accounts dialog box](images/cs-file-browsers-ext-2.png)
-
-5. When you're done, click **Save**, then **Close**.
-
-6. You can now:
+    a. Atmos host name, which is the UKCloud endpoint URL for your selected storage bucket e.g. atmos.cas.frn00006.ukcloud.com.
 
-    - Browse the buckets and files stored in your account
-
-    - Use the toolbar buttons to upload files and folders
+    b. Atmos port - Please enter 443.
 
-    - Double-click items to download them
+    c. Atmos UID - Please enter your selected storage bucket's Atmos access key e.g. a554f348g57f48729c0247r9730e29a1/10-10-10-89431b-1.
 
-    - Access other operations for selected items through context menus
+    d. Atmos secret - Please enter your selected storage bucket's shared secret key.
 
-    ![Atmos for Chrome interface](images/cs-file-browsers-ext-3.png)
+4. Once the information has been added the installer will connect to the bucket and upload the files to the requested folder. Once the script has finished it will give you a URL output that you should copy to your clipboard and keep it safe.
```

Comment:
Suggest:
When you have provided all the required information, the installer will connect to the bucket and upload the files to the requested folder. Once the script has finished, it will output a URL that you should copy to your clipboard and then keep somewhere safe.